package.json

```diff
@@ -13,7 +13,7 @@
     "jscs": "2.7.x",
     "jshint": "2.8.x",
     "mocha": "2.x.x",
-    "ramda": "0.18.x",
+    "ramda": "~0.19.1",
```

Comment:
Yes. In this case we're depending on behaviour which changed between v0.19.0 and v0.19.1, via ramda/ramda#1576. We need to use at least v0.19.1 to have the "uses `R.toString`-like string representations" tests pass.